src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+        super.setArea(Math.PI * Math.pow(radius, 2));
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Name: circle");
+        super.draw();
+        System.out.println("radius " + radius);
+    }
```

Comment:
You should not call a method of the parent class.
It is necessary to override the implementation for each figure
Fix for each extends
```suggestion
    @Override
    public void draw() {
        System.out.println("description of the figure")
    }
```